lib/core/renderer.js

```diff
@@ -528,6 +528,11 @@ export default class Renderer extends Tapable {
       resourceHints = context.renderResourceHints()
       HEAD += resourceHints
     }
+
+    if (this.options.generate && this.options.generate.nuxtContextHook) {
+      context = this.options.generate.nuxtContextHook(context);
```

Comment:
Get rid of this semi-colon to fix travis

Reply:
Done, thanks.